pipeline-runner/R/handle_data.R

```diff
@@ -34,7 +34,8 @@ reload_scdata_from_s3 <- function (s3, pipeline_config, experiment_id) {
     Bucket = bucket,
     Key = paste(experiment_id, "r.rds", sep = "/")
   )
-  obj <- readRDS(gzcon(rawConnection(body)))
+  obj <- readRDS(conn <- gzcon(rawConnection(body)))
```

Comment:
assignments like these in function arguments are not really nice.
```suggestion
  conn <- gzcon(rawConnection(body))
  obj <- readRDS(conn)
```